admin_guide/sysctls.adoc

```diff
@@ -135,31 +135,49 @@ ifdef::openshift-origin[]
 endif::[]
 
 [[setting-sysctls-for-a-pod]]
-== Setting Sysctls for a Pod
+== Setting sysctls for a pod
 
-Sysctls are set on pods using annotations. They apply to all containers in the
-same pod.
+Sysctls are set on pods using the pod's securityContext. The securityContext
+applies to all containers in the same pod.
 
-Here is an example, with different annotations for safe and unsafe sysctls:
+The following example uses the pod securityContext to set a safe sysctl
+`kernel.shm_rmid_forced` and two unsafe sysctls, `net.ipv4.route.min_pmtu` and
+`kernel.msgmax`. There is no distinction between _safe_ and _unsafe_ sysctls in
+the specification.
 
+[WARNING]
+====
+To avoid destabilizing your operating system, modify sysctl parameters only 
+after you understand their effects.
+====
+
+Modify the YAML file that defines the pod and add the securityContext spec, as
+shown in the following example:
+
+[source,yaml]
 ----
 apiVersion: v1
 kind: Pod
 metadata:
   name: sysctl-example
-  annotations:
-    security.alpha.kubernetes.io/sysctls: kernel.shm_rmid_forced=1
-    security.alpha.kubernetes.io/unsafe-sysctls: net.ipv4.route.min_pmtu=1000,kernel.msgmax=1 2 3
 spec:
+  securityContext:
+    sysctls:
+    - name: kernel.shm_rmid_forced
+      value: "0"
+    - name: net.ipv4.route.min_pmtu
+      value: "552"
+    - name: kernel.msgmax
+      value: "65536"
```

Comment:
Why the sysctl values were changed? I'm not against such a change but I'd like to ensure that there is no mistakes.

Reply:
@php-coder, that's what the [K8 doc](https://github.com/kubernetes/website/pull/8804/files#diff-bbdf255eebe738c7c70b93ea3ddcde37R131) used:

```
  securityContext:
    sysctls:
    - name: kernel.shm_rmid_forced
      value: "0"
    - name: net.ipv4.route.min_pmtu
      value: "552"
    - name: kernel.msgmax
      value: "65536"
```

Reply:
Ok; perhaps our example with `kernel.msgmax=1 2 3` was wrong.

Reply:
https://github.com/kubernetes/website/pull/8804#issuecomment-395904849